dpnp/backend/kernels/dpnp_krnl_statistics.cpp

```diff
@@ -1379,12 +1379,12 @@ void func_map_init_statistics(func_map_t& fmap)
 
     fmap[DPNPFuncName::DPNP_FN_COV][eft_INT][eft_INT] = {eft_DBL, (void*)dpnp_cov_default_c<double>};
     fmap[DPNPFuncName::DPNP_FN_COV][eft_LNG][eft_LNG] = {eft_DBL, (void*)dpnp_cov_default_c<double>};
-    fmap[DPNPFuncName::DPNP_FN_COV][eft_FLT][eft_FLT] = {eft_DBL, (void*)dpnp_cov_default_c<double>};
+    fmap[DPNPFuncName::DPNP_FN_COV][eft_FLT][eft_FLT] = {eft_FLT, (void*)dpnp_cov_default_c<float>};
```

Comment:
I guess we need to discuss the change for DPNP_FN_COV with Numba-DPEX command first, since they are impacted by this.
Also there is #1325 opened around that and it would be great to align what exact fix they are expecting from us here.